servers/navigation/navigation_globals.h

```diff
@@ -36,15 +36,14 @@ namespace NavigationDefaults3D {
 
 // To find the polygons edges the vertices are displaced in a grid where
 // each cell has the following cell_size and cell_height.
-constexpr float navmesh_cell_size{ 0.25f }; // Must match ProjectSettings default 3D cell_size and NavigationMesh cell_size.
-constexpr float navmesh_cell_height{ 0.25f }; // Must match ProjectSettings default 3D cell_height and NavigationMesh cell_height.
-constexpr float navmesh_cell_size_min{ 0.01f };
-constexpr auto navmesh_cell_size_hint{ "0.001,100,0.001,or_greater" };
```

Comment:
The hint should be kept here. It needs to stay in sync with the other default property and was placed here to not forget it when it is just buried inside the server method bindings. In fact similar properties should be moved to the globals over time.

Reply:
Alrighty, I've moved it back (but replaced `auto` with an explicit type).